### PR TITLE
added hackathon-starter and django-hackathon-starter

### PR DESCRIPTION
**Pull request by kugiyasan**

### CODE REVIEW

File: submodules.json [hunk 1]
```diff
@@ -133,5 +133,15 @@
     "name": "wordpress-boilerplate",
     "url": "https://github.com/DevinVinson/WordPress-Plugin-Boilerplate",
     "tags": ["wordpress", "website", "wordpress-plugin","web"]
+  },
+  {
+    "name": "hackathon-starter",
+    "url": "https://github.com/sahat/hackathon-starter",
```
Comment (mintbomb27) [inline submodules.json:139]: Hey! This repo already exists at [84](https://github.com/IEEE-VIT/templa-rs/blob/f7d3b665197d4e1bcb8c8aa8c14d56a44d2345c8/submodules.json#L84)?

**Review by mintbomb27 — CHANGES_REQUESTED** [review]
Rest all LGTM!

File: submodules.json (hunk 1 above)
Comment (kugiyasan) [reply submodules.json]: Oops I missed it, I should've double checked first!

**Review by Mannan-Goyal — APPROVED** [review]

**Review by mintbomb27 — APPROVED** [review]
LGTM! Thank you for your contribution!

**Pull request merged**